packages/grid/x-data-grid-pro/src/models/dataGridProProps.ts

```diff
@@ -101,6 +102,16 @@ export interface DataGridProPropsWithDefaultValue extends DataGridPropsWithDefau
    * @returns {boolean} A boolean indicating if the group is expanded.
    */
   isGroupExpandedByDefault?: (node: GridGroupNode) => boolean;
+  /**
+   * If `true`, columns are autosized after the datagrid is mounted.
```

Comment:
```suggestion
   * If `true`, columns are auto-sized after the data grid is mounted.
```

Reply:
I've been using `autosized` and `autosizing` consistently in the comments & docs, I don't have a strong opinion but I have a slight preference for it being a single word.

Same for `datagrid` vs `data grid`, Bilal had the same comment somewhere above [here](https://github.com/mui/mui-x/pull/10180#discussion_r1328139970). For that one, now that I review the docs I see we've been using `data grid` consistently so I'll switch it.

Reply:
This is a straight copy of grammarly suggestion, I haven't looked at which one is actually best.